components/api/src/main/java/com/hotels/styx/api/Url.java

```diff
@@ -121,7 +121,7 @@ public boolean isFullyQualified() {
      * @return true if the URL is absolute.
      */
     public boolean isAbsolute() {
-        return path.startsWith("/");
```

Comment:
What is this method used for? We have changed from _it's an absolute path_ to i_t's an absolute URI_ .  What's the real purpose?

Reply:
I can only guess. I believe the original intention was to determine if the request is in origin-form or absolute-form. My gut feeling is that we don't really need this method in the URL class.

As to why this change? It is because every path component always starts with `/` therefore it always returned true.

Reply:
I would at least mark it as deprecated then. I think from previous discussions our goal could be to completely remove Url from the API.